apis/swagger/agent/core/core/agent.swagger.json

```diff
@@ -349,8 +349,7 @@
           "format": "int64"
         },
         "indexing": {
-          "type": "boolean",
-          "format": "boolean"
```

Comment:
why these swagger files updated?

Reply:
don't know, I just run make update for dependencies update.
may be swagger version ?

Reply:
okay

Reply:
I checked swagger specification, there is not format field for boolean type.
maybe old version has format for boolean type.
FYI: https://swagger.io/specification/#data-types

Reply:
@rinx (mention for FYI) and I'll merge this PR